src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -16,7 +16,7 @@
 error_reporting(-1);
 
 // PHPUnit 4.8 does not support PHP 7, while 5.1 requires PHP 5.6+
-$PHPUNIT_VERSION = PHP_VERSION_ID >= 50600 ? '5.1' : '4.8';
```

Comment:
I suggest using 5.3 rather than 5.1, as 5.1 is EOLed (and then, we should start working on making our testsuite compatible with 5.4+ to use supported PHPunit versions)